navigator_engine/config.py

```diff
@@ -11,11 +11,12 @@ class Config(object):
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     LOGGING_LEVEL = logging.INFO
     DASH_REROUTE_PREFIX = '/engine'
+    DEFAULT_DECISION_GRAPH = f'{base_directory}/Estimates 22 BDG [Final].xlsx'
 
 
 class Testing(Config):
     TESTING = True
-    SQLALCHEMY_DATABASE_URI = f'sqlite+pysqlite:////{base_directory}/../testing.db'
```

Comment:
@jsoppela Why the change here?

Reply:
This was not an intended change, I have reverted it.